install/Knative-custom-install.md

```diff
@@ -66,13 +66,13 @@ service mesh. If you install any of the following options, you must install
 
 #### Istio installation options
 
-| Istio Install Filename     | Description                                                            |
-| -------------------------- | ---------------------------------------------------------------------- |
-| **[`istio-crds.yaml`][a]** | Creates CRDs before installing Istio.                                  |
-| **[`istio.yaml`][b]**      | Install Istio with service mesh enabled (automatic sidecar injection). |
-| [`istio-lean.yaml`][c]     | Install Istio and disable the service mesh by default.                 |
+**Install files in _bold italics_ are the recommended standard install files suitable for most use cases.**
```

Comment:
```suggestion
**The files in _bold italics_ are the recommended standard install files suitable for most use cases.**
```

I originally read "Install files" as Verb-noun, so suggest using an article to make it clearer.